proposals/istio.adoc

```diff
@@ -0,0 +1,90 @@
+== istio
+
+*Name of project*: Istio +
+
+*Description*: +
+
+As monolithic applications are decomposed into microservices, software
+teams have to worry about the challenges inherent in integrating services
+in distributed systems: they must account for service discovery, load
+balancing, fault tolerance, end-to-end monitoring, dynamic routing for
+feature experimentation, and perhaps most important of all, compliance and
+security. +
+
+Istio eases the monolith to microservice transformation and the
+microservice application architecture by systemically injecting proxies
+into the network paths between services. The proxies form a service mesh
+that is dynamically configured and managed by the Istio control
+plane. Istio adds traffic management to microservices and creates a basis
+for value-add capabilities like security, monitoring, routing, connectivity
+management and policy. +
+
+Using the Istio control plane, developers can experiment with different
+features in production, deploy canary releases, and improve the reliability
+of their services without impacting the system as a whole. Operators gain
+fleet-wide visibility through extensive metrics and traces without having
+to rely on developers to emit metrics in their application code. They can
+improve end-to-end security, compliance, and the availability of services
+with minimal disruption to the development process. +
+
+The control plane is agnostic of the underlying orchestration platform
+(e.g., Kubernetes, Mesos, Cloud Foundry, Nomad, or even just VMs and bare
+metal), as well as the proxy (e.g., Envoy, Linkerd, Nginx, etc.). The
+control plane thus creates a basis for platform independent value-add
+capabilities like security, monitoring, routing, connectivity management
+and policy enforcement, in addition to easing migration across cloud
+providers and orchestration platforms. +
+
+In summary, the service mesh weaved and managed by Istio provides a uniform
+platform-independent way to connect, manage, and secure microservices. +
+
+*Sponsor / Advisor from TOC*: Brian Grant +
+*Preferred maturity level*: incubating +
```

Comment:
The incubation stage requires three real deployments of suitable scope:  Reference: https://github.com/cncf/toc/blob/master/process/graduation_criteria.adoc

Might consider the inception stage?  I suspect after 1.0 is released, incubation will be a nobrainer.